Resources/public/js/models/ez-usermodel.js

```diff
@@ -104,6 +133,17 @@ YUI.add('ez-usermodel', function (Y) {
              */
             remoteId: {
                 value: ''
+            },
+
+            /**
+             * The user's avatar
+             *
+             * @attribute avatar
+             * @default {}
```

Comment:
I think you should set it to `null` (both here and in `value` below) to be consistent when the default value in `_parseStruct` and the *empty value* of the Image field. This will also ease the check to detect if there's an avatar or not.